styles/styles.css

```diff
@@ -29,18 +29,41 @@ header p {
   margin: 0;
 }
 
+
+/* Match section content to height of viewport */
+
+
+@media (min-height: 750px) {
+  .storyline {
+    margin: 100px inherit;
+  }
+}
+/*
+@media (max-height: 900px) {
+  .storyline {
+    margin: 700px 0;
+  }
+}
+
+@media (max-height: 1200px) {
+  .storyline  {
+    margin: 1000px 0;
+  }
+} */
+
+/* Format sections of the storyline */
 .storyline {
   display: flex;
 }
 
-.story-scene {
+.story-divider {
+  margin-top: 5%;
   margin-bottom: 20px;
   justify-content: left;
```

Comment:
https://getbootstrap.com/docs/4.1/utilities/flex/#justify-content

The benefit of using class utilities is that you are tying your css layout more directly to your HTML, which usually doesn't get shared.

When you work in HTML, it's faster to position things around.

Reply:
I'll work on using the utilities more often. I used align-items: center in my latest code and it work better than the code I had here.